crates/runtime/tests/models/hf.rs

```diff
@@ -151,6 +170,80 @@ async fn huggingface_test_nsql() -> Result<(), anyhow::Error> {
         "Expected a single record containing the value 10"
     );
 
+    // ensure all spans are exported into task_history
+    let _ = trace_provider.force_flush();
+
+    let tasks = get_executed_tasks(&rt, task_start_time.into()).await?;
```

Comment:
I wonder if we can do a `v1/sql` query and snapshot test the returned JSON value. This will make less code, and more extensible.